### PR TITLE
Set version command output to Stdout

### LINKED ISSUE #1868: version: prints to Stderr instead of StdOut
- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).
- [x] Yes, I've tried with the standalone linter if available. (https://golangci-lint.run/usage/linters/)

<details><summary>Commands print to Stderr instead of StdOut</summary>

Trying to fetch the version in a shell script: `CI_VERSION=$(golangci-lint version --format short)` I noticed the variable was empty and I could see the version printed in the terminal.
After investigating I found out that `golangci-lint` uses Cobra's `Println` function:
https://github.com/golangci/golangci-lint/blob/c1d7cfd721401395e6a454b6af55bfc2dc1089d1/pkg/commands/version.go#L35-L38

According to Cobra, that function prints to `Stderr` per default:
https://github.com/spf13/cobra/blob/b55fa79836c63eab81e884e6ec1d7e4ae5c7f9b5/command.go#L1201

</details>

<details><summary>Version of golangci-lint</summary>
Installed automatically with VS Code

```console
$ golangci-lint --version
golangci-lint has version v1.38.0 built from (unknown, mod sum: "h1:hgZsLRzZrjhpp44Ak+fhXNzgrbDF39ETf22a+Jd3fJQ=") on (unknown)
```

</details>

<details><summary>Config file</summary>

```console
$ cat .golangci.yml
run:
  modules-download-mode: readonly
```

</details>

<details><summary>Go environment</summary>

```console
$ go version && go env
go version go1.16.2 darwin/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="<redacted>/Library/Caches/go-build"
GOENV="<redacted>/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="<redacted>/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="<redacted>"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/Cellar/go/1.16.2/libexec"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.16.2/libexec/pkg/tool/darwin_amd64"
GOVCS=""
GOVERSION="go1.16.2"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/dev/null"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -arch x86_64 -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/8c/qd3ywlk1039f53l5m8mnfg1w0000gn/T/go-build3420486587=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
NOT RELEVANT
```

</details>

<details><summary>Code example or link to a public repository</summary>

```bash
export CI_VERSION=$(golangci-lint version --format short)
echo "current: ${CI_VERSION}"
```

</details>

### PR DESCRIPTION
**Pull request by clebs**
Default command output was set to Stderr, now set to Stdout.

Fixes #1868

### CODE REVIEW

File: pkg/commands/version.go [hunk 1]
```diff
@@ -33,6 +34,7 @@ func (e *Executor) initVersion() {
 		Use:   "version",
 		Short: "Version",
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			cmd.SetOut(os.Stdout)
```
Comment (ldez) [inline pkg/commands/version.go:37]: I think you can just replace `cmd.Println` `fmt.Println`

**Review by ldez — CHANGES_REQUESTED** [review]

File: pkg/commands/version.go (hunk 1 above)
Comment (clebs) [reply pkg/commands/version.go:37]: Sure thing, will do the change.

Comment (clebs) [reply pkg/commands/version.go]: Changed all cobra based prints to `fmt`.

**Review by ldez — APPROVED** [review]
LGTM

**Comment by clebs** [comment]
Thanks for the quick review!

**Pull request merged**